packages/manager/modules/pci/src/projects/creating/controller.js

```diff
@@ -31,7 +31,31 @@ export default class PciProjectCreatingCtrl {
   getDeliveredProjectId() {
     return this.pciProjectCreating
       .getOrderDetails(this.orderId)
-      .then((details) => get(details, '[0].domain'));
+      .then((details) => {
+        const orderItemsDetailPromises = details.map(({ orderDetailId }) =>
+          this.pciProjectCreating
+            .getOrderItemDetails(this.orderId, orderDetailId)
+            .then((data) => {
+              return { orderDetailId, item: data };
+            }),
+        );
+
+        return this.$q
+          .all(orderItemsDetailPromises)
+          .then((orderItemsDetails) => {
+            return { details, orderItemsDetails };
+          });
+      })
+      .then(({ details, orderItemsDetails }) => {
+        const itemDetails = orderItemsDetails.find(
+          ({ item }) => item.order.plan.code === PCI_HDS_ADDON.parentPlanCode,
+        );
+        const detail = details.find(
+          ({ orderDetailId }) => itemDetails.orderDetailId === orderDetailId,
+        );
+
+        return detail.domain;
```

Comment:
Looks like a lot of call for just one information

Reply:
Yes, unfortunately to deduce this info we need to do them.